doc/release-notes/0.40/0.40.md

```diff
@@ -0,0 +1,122 @@
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# Eclipse OpenJ9 version 0.40.0 release notes
+
+These release notes support the [Eclipse OpenJ9&trade; 0.40.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.40.0/plan).
+
+## Supported environments
+
+OpenJ9 release 0.40.0 supports OpenJDK 8, 11, 17 and 20.
+
+All releases are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests provided by Adoptium.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.40.0. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.40/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr>
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/17073">#17073</a></td>
+<td valign="top">Support for persistent shared classes cache is now added on z/OS&reg; systems</td>
+<td valign="top">All versions (z/OS)</td>
+<td valign="top">Persistent shared caches are now supported on z/OS systems. On 31-bit VM on all versions of z/OS, the shared cache is memory mapped within the 0-2 GB address range. On 64-bit VM on z/OS version 2.3 and earlier, the persistent shared cache is memory mapped within the 0-2 GB address range but on z/OS version 2.4 and later the shared cache is memory mapped above the 2 GB address range. </td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/17425">#17425</a></td>
+<td valign="top">New <tt>-XX:codecachetotalMaxRAMPercentage</tt> option is added to set the maximum limit for the total JIT code cache size based on the amount of physical memory available to the VM.</td>
+<td valign="top">All versions</td>
+<td valign="top">The new option is used to set the code cache maximum size limit as a percentage of the available physical memory. In memory constrained environments, the percentage limit prevents the VM from using too much memory for its code caches and thus, leaving too little memory for other VM or JIT data structures.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/7021">#7021</a></td>
```

Comment:
I think this is referring to the wrong issue.  7021 is an OpenJ9 pull request from 2019.  I think this should refer to OMR pull request 7021:  [`https://github.com/eclipse/omr/issues/7021`](https://github.com/eclipse/omr/issues/7021)